maps/nerva/nerva_areas.dm

```diff
@@ -727,4 +727,22 @@
 //ship
 
 /area/ship/combat
-	name = "\improper Ship"
\ No newline at end of file
+	name = "\improper Ship"
+
+// Substations
+
+/area/engineering/substation
```

Comment:
You'll need to add these under nerva_unit_testing.dm as exceptions for vents+scrubbers.